build-system/tasks/storybook/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Gulp storybook tasks",
   "main": "index.js",
   "devDependencies": {
-    "@ampproject/storybook-addon": "1.1.6",
+    "@ampproject/storybook-addon": "^1.1.7",
```

Comment:
Pin dependency.

```suggestion
    "@ampproject/storybook-addon": "1.1.7",
```

Reply:
Thanks for the review! The package.json changes are no longer in this PR since renovate bot beat me to it, so no action here except merging with master.